src/testdir/test_filetype.vim

```diff
@@ -595,6 +595,7 @@ def s:GetFilenameChecks(): dict<list<string>>
     rpgle: ['file.rpgle', 'file.rpgleinc'],
     robot: ['file.robot', 'file.resource'],
     robots: ['robots.txt'],
+    roc: ['file.roc'],
```

Comment:
Why did you add a test for filetype detection without implementing it?

Reply:
Mistake on my part